### PR TITLE
Hacemus

### PR DESCRIPTION
**Pull request by eltsu7**
https://t.me/c/1363070040/153134

### CODE REVIEW

File: teekkari.py [hunk 1]
```diff
@@ -75,18 +75,29 @@ def handleHakemus(self, update: Update, context: CallbackContext):
         # Imneversorry, [16.03.20 14:27]
         # hyy-vä
         bot = context.bot
+
+        # https://t.me/c/1363070040/153134
+        if 'hacemus' in update.message.text.lower():
+            txtHyyvä = 'hy-wae'
```
Comment (Cadiac) [inline teekkari.py:81]:
```suggestion
            txtHyyva = 'hy-wae'
```

File: teekkari.py [hunk 2]
```diff
@@ -75,18 +75,29 @@ def handleHakemus(self, update: Update, context: CallbackContext):
...
+            txtTapanSut = 'i cill u'
+            txtTapanKaikki = 'HEADSHOT'
+        else:
+            txtHyyvä = 'hyy-vä'
```
Comment (Cadiac) [inline teekkari.py:85]:
```suggestion
            txtHyyva = 'hyy-vä'
```

File: teekkari.py [hunk 3]
```diff
@@ -75,18 +75,29 @@ def handleHakemus(self, update: Update, context: CallbackContext):
...
+            txtTapanSut = 'tapan sut'
+            txtTapanKaikki = 'TAPAN KAIKKI'
+
         if random.randint(0, 9) == 0 and (update.message.from_user.id != 153013548 or random.randint(0, 3) == 0):
             if random.randint(0, 200) == 0:
                 bot.sendSticker(chat_id=update.message.chat_id, sticker='CAADBAADJgADiR7LDbglwFauETpzFgQ')
             else:
-                bot.sendMessage(chat_id=update.message.chat_id, text='hyy-vä')
+                bot.sendMessage(chat_id=update.message.chat_id, text=txtHyyvä)
```
Comment (Cadiac) [inline teekkari.py:93]:
```suggestion
                bot.sendMessage(chat_id=update.message.chat_id, text=txtHyyva)
```

**Review by Cadiac — CHANGES_REQUESTED** [review]

**Review by Cadiac — APPROVED** [review]
Looks guud to me :DDD

**Review by Aivoapina — APPROVED** [review]

**Pull request merged**